### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by Therealadetunji**
Add HTML, CSS files for Hello microverse project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (danmainah) [inline README.md:34]: - [ ] [OPTIONAL] It would be nice to add more information to your readme file to make it look more professional.

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi  @Therealadetunji, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] Kindly make sure your pull request title is descriptive and also while at it add a pull request description with information about what you did in this pull request.
- [ ] Kindly make sure to set up your linters correctly and ensure all checks are passing.
- [ ] Kindly ensure to add the .gitignore file, in it makes sure test.md file and node_modules directory are ignored.

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md (hunk 1 above)
Comment (Therealadetunji) [reply README.md:34]: Thank you for the feedback
i noticed you flagged my css file and html file...what exactly is the issue?
is it the way i constructed it? or i used it wrongly?

Comment (Therealadetunji) [reply README.md:34]: @danmainah

Comment (Therealadetunji) [reply README.md:34]: Thank you for the feedback
i noticed you flagged my css file and html file...what exactly is the issue?
is it the way i constructed it? or i used it wrongly?

@danmainah

**Review by phelian23 — CHANGES_REQUESTED** [review]
Hi @Therealadetunji, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
- [ ] As requested by the previous reviewer, kindly add the linters file correctly. Follow the instructions [here](https://github.com/microverseinc/linters-config) to set up the linters correctly.
- [ ]  Kindly add some description to the pull request, summarizing what changes made in the pull request.
Capture


_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @Therealadetunji, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [ ] Kindly add the Pull Request title and description. As shown in the below video attached, kindly change `Newbranch1` to `Hello Microverse Project`. 
- [ ] Also, edit the section where its written No description provided, kindly edit is clicking the 3 dots shown. You can add for description something like`Add HTML, CSS files for hello microverse project.`

https://user-images.githubusercontent.com/70269507/170207296-fd75435a-bf09-46ff-bb8d-97f5f7ac2b4f.mov

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by m05e5 — APPROVED** [review]
# Hi @Therealadetunji , 

### Status: Approved ✔️
goodjob


:+1: You have done all the required changes asked by the previous reviewer. Well Done.

Your project is complete! 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

-_(optional)_ Please next time kindly give a descriptive branch name
**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**